imap_processing/utils.py

```diff
@@ -122,8 +127,21 @@ def create_dataset(packets: list[Packet], met_name="shcoarse") -> xr.Dataset:
     description_dict = {}
 
     for data_packet in packets:
+        current_data = (
```

Comment:
```suggestion
        data_to_include = (
```

I was a bit confused at first what `current` meant in this context. I think it is fine to leave as is but just wanted to offer this suggestion.